experimental/dds/attributable-map/src/mapKernel.ts

```diff
@@ -358,6 +356,7 @@ export class AttributableMapKernel {
 
 		// Clear the data locally first.
 		this.clearCore(true);
+		this.clearAllAttribution();
```

Comment:
this was only happening in the applyStashedOps path, but i think it should happen whenever there is a local clear.

@clarenceli-msft @Abe27342